modules/http2/h2_c2_filter.c

```diff
@@ -668,147 +668,13 @@ apr_status_t h2_c2_filter_response_out(ap_filter_t *f, apr_bucket_brigade *bb)
 }
 
 
```

Comment:
Removing all the pain in HTTP/2 that needed to emulate `Transfer-Encoding: chunked` for request bodies without `Content-Length`, so that `HTTP_IN` could parse it and no one assumed there was no body.